pennylane/tape/tape.py

```diff
@@ -456,6 +446,7 @@ def __exit__(self, exception_type, exception_value, traceback):
         QuantumTape._lock.release()
         self._process_queue()
         self._trainable_params = None
+        self._update()
```

Comment:
Actually, now I see that `_process_queue` does call `_update`.  So we might not need to manually call it again here.

Reply:
You are right, thank you!